inertia.go

```diff
@@ -34,9 +65,16 @@ type page struct {
 	Version   string                 `json:"version"`
 }
 
-func (i *Inertia) render(w http.ResponseWriter, r *http.Request, componentName string, props P) {
+func (i *Inertia) render(w http.ResponseWriter, r *http.Request, componentName string, p P) {
+
+	props := P{}
+	props.merge(i.shared)
+
+	for prop, lazy := range i.sharedLazy {
+		props[prop] = lazy(r)
+	}
```

Comment:
The laziness of the shared props is useless since they're evaluated on every request. They should be evaluated below, where regular lazy props are evaluated.

Reply:
See latest commit 👍